# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/RangeUtils.cpp
# Commit: 4ee2d1219a0a
# Full Hash: 4ee2d1219a0a852d740e43e8421c0cc3e1cd9c4e
# Author: Sean Feng <sefeng@mozilla.com>
# Date: 2025-05-10 09:29:34
# Regressor Bug: 1932150
# File Overlap Count: 1
# Description:
#   Bug 1932150 - Update ranges to correctly set and update nodes for flattened selection r=jjaschke,smaug,dom-core
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D231589
# ==============================================================================

diff -r 29195ee87cf4 -r 4ee2d1219a0a dom/base/RangeUtils.cpp
--- a/dom/base/RangeUtils.cpp	Fri May 09 17:16:36 2025 +0000
+++ b/dom/base/RangeUtils.cpp	Fri May 09 17:16:37 2025 +0000
@@ -10,6 +10,7 @@
 #include "mozilla/dom/AbstractRange.h"
 #include "mozilla/dom/Document.h"
 #include "mozilla/dom/ShadowRoot.h"
+#include "mozilla/dom/HTMLSlotElement.h"
 #include "nsContentUtils.h"
 #include "nsFrameSelection.h"
 